packages/material/src/complex/MaterialTableControl.tsx

```diff
@@ -109,6 +114,7 @@ const generateCells = (
       schema,
       rowPath,
       cellPath: rowPath,
+      title: schema.title ? schema.title : '',
```

Comment:
```suggestion
      title
```

Reply:
The variable 'title' that we declared in the conditional above is out of scope here. But maybe props that aren't objects don't need titles?